demonstrations/tutorial_apply_qsvt.py

```diff
@@ -288,16 +288,17 @@ def loss_func(phi):
 #
 # .. math::
 #
-#    \hat{U}_{real}(\vec{\phi}) = \frac{1}{2} \ ( \hat{U}_{qsvt}(\vec{\phi}) + \hat{U}^{\dagger}_{qsvt}(\vec{\phi}) ).
+#    \hat{U}_{real}(\vec{\phi}) = \frac{1}{2} \ ( \hat{U}_{qsvt}(\vec{\phi}) + \hat{U}^{*}_{qsvt}(\vec{\phi}) ).
 #
-# Here we use a two-term LCU to define the quantum function for this operator:
+# Here we use a two-term LCU to define the quantum function for this operator. We obtain the complex
+# conjugate of :math:`\hat{U}_{qsvt}` by taking the adjoint of the operator block-encoding :math:`A^{T}`:
 
 
 def real_u(A, phi):
     qml.Hadamard(wires="ancilla1")
 
     qml.ctrl(sum_even_odd_circ, control=("ancilla1",), control_values=(0,))(A, phi, "ancilla2", [0, 1, 2])
-    qml.ctrl(qml.adjoint(sum_even_odd_circ), control=("ancilla1",), control_values=(1,))(A, phi, "ancilla2", [0, 1, 2])
+    qml.ctrl(qml.adjoint(sum_even_odd_circ), control=("ancilla1",), control_values=(1,))(A.T, phi, "ancilla2", [0, 1, 2])
```

Comment:
From my point of view with the idea of "replicating in PL what we wrote with pen and paper", it makes more sense to introduce the conjugate matrix directly instead of making the transpose of the conjugate.  Is there any other reason to do it as the transpose of the adjoint?

```suggestion
    qml.ctrl(sum_even_odd_circ, control=("ancilla1",), control_values=(1,))(np.conjugate(A), phi, "ancilla2", [0, 1, 2])
```

If you accept this change. The added paragraph would have to be modified

Reply:
I have no problem with this 👍🏼

Reply:
Updated!